src/app/admin/subject/[slug]/page.tsx

```diff
@@ -14,11 +14,11 @@ import { Input } from "@/components/ui/input";
 import { useUser } from "@/components/hooks/UserContext";
 import { db } from "@/lib/firebase";
 import { doc, getDoc, setDoc } from "firebase/firestore";
-import { Link } from "../_components/link";
-import apClassesData from "@/app/admin/apClasses.json";
+import Link from "next/link";
```

Comment:
This needs to be from the custom `link.tsx` component for the navigation blocker to work on Link components.

Reply:
Ok so just replace Link from "next/link" to { Link }?